From bc82b45dcfe69d876dc312c16db4361b8a7366c7 Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Sun, 12 Sep 2021 12:24:44 +0300
Subject: [PATCH] [SW-55224] gaudi2: move sdram_base_full to common

Move SDRAM_BASE_FULL define to main common header,
instead of arm common header.

Change-Id: I63ae2d0d16f386b2fcebacf89f116b0e045e9534
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/arch/gaudi2/arm/common.h b/arch/gaudi2/arm/common.h
index aa7954d..da1e1f9 100644
--- a/arch/gaudi2/arm/common.h
+++ b/arch/gaudi2/arm/common.h
@@ -20,11 +20,9 @@
 #ifdef TARGET_IS_ONIX
 #define PPBOOT_TEXT_BASE		0x8200000000
 #define SDRAM_BASE			0x8200000000ull
-#define SDRAM_BASE_FULL			0x1001000200000000ull
 #else
 #define PPBOOT_TEXT_BASE		0x7FFD000000
 #define SDRAM_BASE			0x7FFD000000ull
-#define SDRAM_BASE_FULL			0x1000FFFFFD000000ull
 #endif
 
 #define HL_FW_BOOT_PTRS_BASE		0x80000
diff --git a/arch/gaudi2/common/common.h b/arch/gaudi2/common/common.h
index 79eaa5f..f428cf8 100644
--- a/arch/gaudi2/common/common.h
+++ b/arch/gaudi2/common/common.h
@@ -23,6 +23,7 @@
 	#define DECODER_ENABLED_MASK 0x1AB
 	#define NIC_PORTS_MASK 0x0
 	#define TPC_ENABLED_MASK 0x1020101
+	#define SDRAM_BASE_FULL 0x1000FFFFFD000000ull
 #elif TARGET_IS_ONIX
 	#define MME_MASK 0
 	#define EDMA_ENABLED_MASK 0x0
@@ -30,6 +31,7 @@
 	#define DECODER_ENABLED_MASK 0x1AB
 	#define NIC_PORTS_MASK 0
 	#define TPC_ENABLED_MASK 0x1020101
+	#define SDRAM_BASE_FULL	0x1001000200000000ull
 #else /* ASIC */
 	#define MME_MASK 0xF
 	#define EDMA_ENABLED_MASK 0xFF
@@ -37,6 +39,7 @@
 	#define DECODER_ENABLED_MASK 0x1FF
 	#define NIC_PORTS_MASK 0xFFFFFF
 	#define TPC_ENABLED_MASK 0x1FFFFFF
+	#define SDRAM_BASE_FULL 0x1000FFFFFD000000ull
 #endif
 
 #define NUM_OF_DCORES			4
